<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMain</class>
 <widget class="QWidget" name="AdminMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidgetAdmin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>590</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>139</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnCrudAdmin">
        <property name="text">
         <string>ADMINISTRADOR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCrudUsuarios">
        <property name="text">
         <string>USUARIO</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>185</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Administracion &gt; Principal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
